test/parallel/test-console-sync-write-error.js

```diff
@@ -0,0 +1,17 @@
+'use strict';
```

Comment:
It would be helpful to have additional tests that verify this works with a variety of stdout/stderr configurations (e.g. stdio as a file, stdio as a pipe, etc)